target/linux/ramips/image/mt7621.mk

```diff
@@ -222,6 +222,15 @@ define Device/buffalo_wsr-600dhp
 endef
 TARGET_DEVICES += buffalo_wsr-600dhp
 
+define Device/cudy_wr1300
+  IMAGE_SIZE := 15872k
+  DEVICE_VENDOR := Cudy
+  DEVICE_MODEL := WR1300
+  DEVICE_PACKAGES := kmod-mt76 kmod-usb2 kmod-usb3 kmod-usb-ledtrig-usbport
```

Comment:
You need to use a specific package instead of kmod-mt76 ...

Reply:
Added both of the wireless modules explicitly.